source/http_connection.c

```diff
@@ -222,23 +278,18 @@ napi_value aws_napi_http_connection_close(napi_env env, napi_callback_info info)
         return NULL;
     }
 
-    struct http_nodejs_connection *node_connection = NULL;
-    if (napi_get_value_external(env, node_args[0], (void **)&node_connection)) {
-        napi_throw_error(env, NULL, "Failed to extract http_nodejs_connection from external");
+    struct http_connection_binding *binding = NULL;
+    if (napi_get_value_external(env, node_args[0], (void **)&binding)) {
+        napi_throw_error(env, NULL, "Failed to extract http_connection_binding from external");
         return NULL;
     }
 
-    if (node_connection->connection) {
-        aws_http_connection_close(node_connection->connection);
+    if (binding->connection) {
+        aws_http_connection_close(binding->connection);
+        aws_http_connection_release(binding->connection);
```

Comment:
trivial: don't need to call `aws_http_connection_close()` release() does that for you

Reply:
I've fixed this in a follow-up